### PR TITLE
@broskoski: Artwork#related

### PR DESCRIPTION
**Pull request by dzucconi**

### CODE REVIEW

File: test/schema/artwork/index.js [hunk 1]
```diff
@@ -137,4 +138,50 @@ describe('Artwork type', () => {
...
+      `;
+
+      return graphql(schema, query)
+        .then(({ data }) => {
+          data.artwork.banner.should.eql({
+            __typename: 'RelatedSale',
+            name: 'Y2K',
+            href: '/auction/existy',
+            end_at: '31:12:1999',
+          });
```
Comment (dzucconi) [inline test/schema/artwork/index.js]: Makes the `BannerModule` very simple/clear

Comment (broskoski) [inline test/schema/artwork/index.js]: Daaamn, a whole new world.

File: schema/artwork/index.js [hunk 2]
```diff
@@ -120,6 +122,21 @@ const ArtworkType = new GraphQLObjectType({
...
+      related: {
+        type: RelatedType,
+        description: 'Returns the associated Fair or Sale',
+        resolve: ({ id }) => {
+          const options = { artwork: [id], active: true, size: 1 };
+          return Promise.all([
+            gravity('related/fairs', options),
+            gravity('related/sales', options),
+          ]).then(([fairs, sales]) => {
+            fairs.map(fair => fair.related_type = 'Fair'); // eslint-disable-line no-param-reassign
```
Comment (craigspaeth) [inline schema/artwork/index.js]: `// eslint-disable-line no-param-reassign` :unamused:

Comment (dzucconi) [inline schema/artwork/index.js]: Yeah; I guess i should be doing a `_.assign({}, fair, { related_type: 'Fair' })` :palm_tree:

**Pull request merged**